frontend/app/src/components/molecules/nav-bar/banner.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Button } from '@/components/ui/button';
+
+export interface BannerProps {
+  message: React.ReactNode;
```

Comment:
qq: can we make this a string?

Reply:
i think we may want to include an icon or additional links here (i.e. link to migration docs)